Libraries/Boards/MAX32662/FTHR/examples.txt

```diff
@@ -0,0 +1,43 @@
+ADC
+ARM-DSP/arm_bayes_example
+ARM-DSP/arm_class_marks_example
+ARM-DSP/arm_convolution_example
+ARM-DSP/arm_dotproduct_example_f32
+ARM-DSP/arm_fft_bin_example
+ARM-DSP/arm_fir_example
+ARM-DSP/arm_graphic_equalizer_example
+ARM-DSP/arm_linear_interp_example
+ARM-DSP/arm_matrix_example
+ARM-DSP/arm_signal_converge_example
+ARM-DSP/arm_sin_cos_example
+ARM-DSP/arm_svm_example
+ARM-DSP/arm_variance_example
+Bootloader_Host
```

Comment:
Please remove
- Bootloader_Host
- EEPROM_Emulator
- I2C_Manager  (if you validated it is ok)

Reply:
done [1a9774e](https://github.com/Analog-Devices-MSDK/msdk/pull/469/commits/1a9774e271546f20caceccba8c82020602e777b1)

Note : I2C_Manager is skipped. Please check confluence.